### DISCUSSION TITLE
It just stop working on all of app that have it for some reason

### DISCUSSION THREAD

**Discussion by GTAandApplechannel** [Anarios/return-youtube-dislike#813] [Q&A]
It just stop working on all of app that have it for some reason on (YouTube Vanced - YouTube ReVanced, Kiwi Browser usin
![Screenshot_20221106-145313](https://user-images.githubusercontent.com/117007621/200160215-87e9db6e-fe6f-4d6e-bfe9-27ef87fb9048.png)
![Screenshot_20221106-145338](https://user-images.githubusercontent.com/117007621/200160219-830b6787-d44e-475f-a0d9-03ed0ce99eb6.png)
![Screenshot_20221106-145536](https://user-images.githubusercontent.com/117007621/200160222-1c411140-9024-4ed3-b2af-838562f925aa.png)
g Google Chrome extension)

**Comment by ImpenetrableNoble** [comment]
I agree it suddenly got removed, so I don't know either.
